scripts/linux/publishReleasePackages.sh

```diff
@@ -51,8 +51,11 @@ check_os() {
     elif [[ "$PACKAGE_OS" == "redhat8" ]]; then
         OS_NAME="redhat"
         OS_VERSION="8"
+    elif [[ "$PACKAGE_OS" == "windows" ]]; then
```

Comment:
I'd prefer that we _not_ add Windows here. If someone passes "windows" to this script, it should fail with an "unsupported" error.